cdc/generation.hh

```diff
@@ -155,4 +155,13 @@ future<utils::chunked_vector<mutation>> get_cdc_generation_mutations_v2(
     schema_ptr, utils::UUID gen_uuid, const cdc::topology_description&,
     size_t mutation_size_threshold, api::timestamp_type mutation_timestamp);
 
+// Translates the CDC generation data given by a `cdc::topology_description` into a vector of mutations,
+// using `mutation_size_threshold` to decide on the mutation sizes. All mutations have the same partition
+// key - "cdc_generation". The timestamp of each cell in each mutation is given by `mutation_timestamp`.
```

Comment:
In the spirit of the comment for `get_cdc_generation_mutations_v2`, I would also explain how `gen_uuid` is used in the mutations (i.e. it's the first clustering key column). No need to say about `"cdc_generation"` partition key since it's unrelated to the parameters of the function and mostly an implementation detail (we have no much better way to represent single-partition tables than to invent an arbitrary partition key).

Reply:
Done.